nodejs/week1/avg.js

```diff
@@ -0,0 +1,14 @@
+const newArguments = process.argv.slice(2);
+if (newArguments.length === 0) {
+  console.log("Provide some nembers");
+} else {
+  // convert the arguments to numbers using function Number and filter the non numbers
+  const numbers = newArguments.map(Number).filter((num) => !isNaN(num));
+  if (numbers.length === 0) {
+    console.log("Provide some valid numbers");
```

Comment:
Well, good job :) 
But just filtering out the characters might not be the case. But what I would suggest that alerting the user to know that he/she has made a mistake, will be a good option.